cluster-autoscaler/README.md

```diff
@@ -21,6 +21,12 @@ there is a big chance that it won't work as expected.
 
 # Notable changes:
 
+CA Version 0.5.2:
+* Fix problems with pods using persisten volume claims in scale up.
```

Comment:
1. s/persisten/persistent
2. I would link the original issue here (https://github.com/kubernetes/contrib/issues/2507).
3. The description is very vague. What do you think about something along the lines of 'Fix a bug causing pods using persistent volume claims to be ignored in scale up consideration'?